backend/rdf4j/src/test/java/org/dotwebstack/framework/backend/rdf4j/Rdf4jModuleTest.java

```diff
@@ -1025,4 +1026,18 @@ void graphQlQuery_returnsBeer_ForBlankNodeMapping() {
     assertThat(castToMap(castToMap(data.get("beer")).get("inspiredBy")).containsKey("person"), equalTo(true));
     assertThat(castToMap(castToMap(data.get("beer")).get("inspiredBy")).get("person"), equalTo("Erik Bierhof"));
   }
+
+  private void assertContainsObject(Model model, String object) {
+    assertThat(model.objects()
```

Comment:
Deze is beter denk ik, er wordt geen string gemaakt en containsString is niet nauwkeurig ivm substrings.
object=brewery1 wordt bv ook gevonden als object=brewery123
```suggestion
     assertTrue(model.objects()
        .stream().anyMatch(o->Objects.equals(o.stringValue(),object)));
```